core/object/ref_counted.cpp

```diff
@@ -48,6 +48,7 @@ void RefCounted::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("init_ref"), &RefCounted::init_ref);
 	ClassDB::bind_method(D_METHOD("reference"), &RefCounted::reference);
 	ClassDB::bind_method(D_METHOD("unreference"), &RefCounted::unreference);
+	ClassDB::bind_method(D_METHOD("get_reference_count"), &RefCounted::reference_get_count);
```

Comment:
I would also rename the C++ method for consistency (and then amend the commit message accordingly).